ansible/roles/ocp4-workload-ccnrd/tasks/post_workload.yml

```diff
@@ -15,6 +15,12 @@
     - "user.info: http://get-a-username-labs-infra.{{ route_subdomain }}"
     - "user.info: "
     - "user.info: You should share this URL (or a shortlink for it) -- It is all they will need to get started!"
+    - "user.info: "
```

Comment:
Not required, but I'd like to see this old way updated to the new way if you are already updating. You can use the `agnosticd_user_info` module to print out `user.info` and preserve functionality today and also save the same as data that can be used in bookbag/Babylon eventually.

https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp4-disconnected-osp-lab/post_software.yml#L10-L39

Reply:
Thanks for the tip, will do next time I'm in here - what are the allowed keys for the `agnosticd_user_info.data` element?

Reply:
`msg`
`data`
`user`

https://github.com/redhat-cop/agnosticd/blob/development/ansible/library/agnosticd_user_info.py

https://github.com/redhat-cop/agnosticd/blob/development/ansible/action_plugins/agnosticd_user_info.py